packages/terra-icon/CHANGELOG.md

```diff
@@ -1,8 +1,17 @@
 Changelog
 =========
+### Added
+* Added `--terra-icon-abnormal-low-light-path-1-fill` variable in IconAbnormalLowLight.module.scss
+
+### Changed
+* IconAbnormalLowLight inline style moved to scss file
 
 Unreleased
 ----------
+### Changed
+* Updated csvtojson devdependency to v2.0.8
+* Updated jsdom devdependency to v15.1.1
+* Updated svgo devdependecy to v1.2.2
```

Comment:
A few of the entries look like they were added above the Unreleased section

Reply:
Updated! [95ea15f](https://github.com/cerner/terra-core/pull/2451/commits/95ea15fb56241f1bdd26df855aee6a5fd8190480)